### PR TITLE
ADOPTERS: direct readers to the website page

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to https://github.com/open-telemetry/opentelemetry.io/issues/2947
- Replaces page content with a note that the page has moved, and a link to its new home.
- Depends on https://github.com/open-telemetry/opentelemetry.io/pull/2961

### CODE REVIEW

**Comment by chalin** [comment]
Link checking will pass after https://github.com/open-telemetry/opentelemetry.io/pull/2961 is merged.

File: ADOPTERS.md [hunk 1]
```diff
@@ -1,32 +1,5 @@
 # OpenTelemetry Adopters
 
-A non-exhaustive, alphabetized list of organizations that have adopted OpenTelemetry.
+This page has moved, see [Adopters].
```
Comment (yurishkuro) [inline ADOPTERS.md:3]:
```suggestion
This page has moved [to the opentelemetry.io website][adopters].
```

**Review by yurishkuro — APPROVED** [review]

File: ADOPTERS.md (hunk 1 above)
Comment (chalin) [reply ADOPTERS.md:3]: That link text isn't semantically meaningful as a representation of the link target. How about the following:

```suggestion
This page has moved to [opentelemetry.io], see [Adopters].

[adopters]: https://opentelemetry.io/ecosystem/adopters/
[opentelemetry.io]: https://opentelemetry.io
```

**Comment by chalin** [comment]
Link checking is passing now 🟢

**Review by trask — APPROVED** [review]

**Pull request merged**